src/main/java/com/twilio/accountsecurity/services/PhoneVerificationService.java

```diff
@@ -35,7 +35,7 @@ public void start(String phoneNumber, String via) {
     }
 
     public VerificationCheckCreator getVerificationCheckCreator(String verificationSid, String phoneNumber, String token) {
```

Comment:
@lopenchi Can the `String token` be removed, too?
```suggestion
    public VerificationCheckCreator getVerificationCheckCreator(String verificationSid, String phoneNumber) {
```